sktime/forecasting/compose/_networkpipeline.py

```diff
@@ -0,0 +1,267 @@
+# -*- coding: utf-8 -*-
+from sktime.forecasting.base._sktime import _SktimeForecaster
+from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
+from sktime.base import _HeterogenousMetaEstimator
+from sktime.transformations.base import _SeriesToSeriesTransformer
+from sktime.utils.validation.forecasting import check_y, check_X
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+
+
+__author__ = ["Viktor Kazakov"]
+__all__ = ["NetworkPipelineForecaster"]
+
+
+class NetworkPipelineForecaster(
+    _OptionalForecastingHorizonMixin,
+    _SktimeForecaster,
+    _HeterogenousMetaEstimator,
+    _SeriesToSeriesTransformer,
```

Comment:
if you inherit from `_SeriesToSeriesTransformer` you have to implement `transform`,  
take a look at `TransformedTargetForecaster` for an example, but this may cause some issues still, so perhaps easiest to start without it being a transformer